Tasks/AzureRmWebAppDeployment/AzureUtility.ps1

```diff
@@ -36,6 +36,22 @@ function Get-AzureRMWebAppDetails
     return $azureRMWebAppDetails
 }
 
+function Get-AzureRMWebAppPublishUrl
+{
+    param([String][Parameter(Mandatory=$true)] $webAppName,
+          [String][Parameter(Mandatory=$true)] $deployToSlotFlag,
+          [String][Parameter(Mandatory=$false)] $resourceGroupName,
+          [String][Parameter(Mandatory=$false)] $slotName)
+
+    Write-Verbose "`t Getting azureRM WebApp Url for web app :'$webAppName'."
+    $azureRMWebAppProperties = Get-AzureRMWebAppPublishUrlARM -webAppName $WebAppName -deployToSlotFlag $DeployToSlotFlag `
+                         -resourceGroupName $ResourceGroupName -slotName $SlotName
+    Write-Verbose "`t Got azureRM azureRM WebApp Url for web app :'$webAppName'."
+
+    Write-Verbose ($azureRMWebAppProperties | Format-List | Out-String)
+    return $azureRMWebAppProperties
```

Comment:
the variable says webAppProperties... I assume we are not returning all the web app properties but only the Webapp url...